saagieapi/apps/apps.py

```diff
@@ -18,27 +19,32 @@ def list_for_project(self, project_id: str) -> Dict:
         ----------
         project_id : str
             UUID of your project (see README on how to find it)
+
```

Comment:
Add `pprint_result: Optional[bool] = True` as parameter ?

Reply:
not needed as this function is not called by other functions